frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js

```diff
@@ -328,6 +333,12 @@ class PlaneController extends React.PureComponent<Props> {
     };
   }
 
+  getLoopedKeyboardControls() {
+    return this.props.tracing.skeleton != null
```

Comment:
```suggestion
    // Note that this code needs to be adapted in case the volumeController also starts to expose
    // looped keyboard controls. For the hybrid case, these two controls would need t be combined then.
    return this.props.tracing.skeleton != null
```